content/cloud-computing/cloud-computing.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Cloud Computing'
+Description: 'Cloud computing refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cloud-computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Cloud computing** refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user. This is becoming more popular with businesses, and has even become ubiquitous in personal computing. Whenever a person sets up a website on GoDaddy, plays World of Warcraft, or streams a movie on Netflix, they are using a cloud computing solution.
+
+## How Cloud Computing Works
+
+The cloud computing service is divided into two sections, [front-end](https://www.codecademy.com/resources/docs/general/front-end) and [back-end](https://www.codecademy.com/resources/docs/general/back-end), that are connected through the Internet. The front-end consists of the client’s computer or computer network. On the back-end, the cloud is made of the following:
+
+- Computers
+- [Servers](https://www.codecademy.com/resources/docs/general/server)
+- [Data Storage Systems](https://www.codecademy.com/resources/docs/general/database)
+
+A cloud computing system can implement any computer program, from data processing to video games. Generally, each application will have its dedicated server.
```

Comment:
Again, this seems to refer more to cloud computing as a specific application rather than a more general concept.